LICENSE-3rdparty.csv

```diff
@@ -1,1408 +1,1408 @@
+core,github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/generator,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/gengateway,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/httprule,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger/genswagger,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger/options,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/runtime,BSD-3-Clause,"Gengo, Inc"
+core,github.com/grpc-ecosystem/grpc-gateway/utilities,BSD-3-Clause,"Gengo, Inc"
+core,github.com/h2non/filetype,MIT,Tomas Aparicio
+core,github.com/h2non/filetype/matchers,MIT,Tomas Aparicio
+core,github.com/h2non/filetype/matchers/isobmff,MIT,Tomas Aparicio
+core,github.com/h2non/filetype/types,MIT,Tomas Aparicio
+core,github.com/hashicorp/consul/api,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/errwrap,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/go-cleanhttp,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/go-hclog,MIT,HashiCorp
+core,github.com/hashicorp/go-immutable-radix,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/go-multierror,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/go-rootcerts,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/golang-lru,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/golang-lru/simplelru,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/hcl/ast,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/hcl/parser,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/hcl/printer,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/hcl/scanner,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/hcl/strconv,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/hcl/token,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/json/parser,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/json/scanner,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/hcl/json/token,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/serf/coordinate,MPL-2.0,UNKNOWN
+core,github.com/hectane/go-acl,MIT,Nathan Osman
+core,github.com/hectane/go-acl/api,MIT,Nathan Osman
+core,github.com/huandu/xstrings,MIT,Huan Du
+core,github.com/ianlancetaylor/cgosymbolizer,BSD-3-Clause,"Free Software Foundation, Inc | The Go Authors"
+core,github.com/imdario/mergo,BSD-3-Clause,Dario Castañé | The Go Authors
+core,github.com/inconshreveable/mousetrap,Apache-2.0,Alan Shreve
+core,github.com/iovisor/gobpf/pkg/cpupossible,Apache-2.0,Kinvolk | PLUMgrid
+core,github.com/iovisor/gobpf/pkg/cpurange,Apache-2.0,Kinvolk | PLUMgrid
+core,github.com/itchyny/gojq,MIT,itchyny
+core,github.com/itchyny/timefmt-go,MIT,itchyny
+core,github.com/jlaffaye/ftp,ISC,Julien Laffaye <jlaffaye@FreeBSD.org>
+core,github.com/jmespath/go-jmespath,Apache-2.0,James Saryerwinnie
+core,github.com/josharian/intern,MIT,UNKNOWN
```

Comment:
Josh Bleecher Snyder

Reply:
Ah, I see that's in `license.md`, one of the filenames I hadn't anticipated!  I added it and this appeared automatically :)